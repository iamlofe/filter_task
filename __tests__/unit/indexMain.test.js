import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';

import IndexComponent from 'index';

describe('App', () => {
    it('should shallow componetn without craching', () => {
        expect(JSON.stringify(IndexComponent)).toMatchSnapshot();
    });
});
