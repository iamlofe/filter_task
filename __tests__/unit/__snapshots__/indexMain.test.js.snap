// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should shallow componetn without craching 1`] = `"{}"`;
